src/foam/nanos/http/FileServlet.java

```diff
@@ -47,9 +47,13 @@ private void fileNotFoundError(HttpServletResponse resp, String file) {
 
   @Override
   protected void doGet(HttpServletRequest req, HttpServletResponse resp) {
-    String pathInfo = req.getPathInfo();
+    String pathInfo;
+    try {
+      pathInfo = req.getRequestURI();
+    } catch (Exception e) {
```

Comment:
Does this help: 
https://github.com/foam-framework/foam2/commit/d5b8a094d1e5d30548ecc5ee70634928b78ee83f